WebRoot/js/zimbraAdmin/accounts/view/ZaAccountMemberOfListView.js

```diff
@@ -751,19 +751,23 @@ ZaAccountMemberOfListView.prototype._setNoResultsHtml = function() {
 	var msg = "";
 	if (this.getCurrentListId().indexOf(ZaAccount.A2_indirectMemberList) >= 0) {
 		var instance = this.parent && this.parent.getInstance();
+		//For Account level
 		if (instance && instance.memberOfLoaded) {
 			msg = ZaMsg.Account_Group_NoInDirectMember;
 		}
+		//For DL
 		else {
-			msg = ZaMsg.MSG_HomeLoading;
+			msg = ZaMsg.Account_Group_NoInDirectMember;
```

Comment:
It can be actually, but I intentionally kept that just for the case if we need to have specific messages at account/dl level. Added comments as well to the block to help on that.